base/multidimensional.jl

```diff
@@ -207,11 +207,11 @@ function _unsafe_getindex(::LinearIndexing, src::AbstractArray, I::AbstractArray
     D = eachindex(dest)
     Ds = start(D)
     s = 0
-    for b in eachindex(I)
+    @inbounds for b in eachindex(I)
```

Comment:
`next` will be affected by the `@inbounds` if it is inlined, since it is only one level deep. If that's okay, then decorating the entire loop is fine.